app/jobs/email_checkin_reminder_job.rb

```diff
@@ -0,0 +1,30 @@
+class EmailCheckinReminderJob < ReservationJob
+  include EmailJobHelper
+  queue_as :default
```

Comment:
it actually only needs to be in `ReservationJob` and jobs that don't inherit from it